source/jsonnet/england-wales/individual/blocks/identity-and-health/other_national_identities.jsonnet

```diff
@@ -19,7 +19,7 @@ local question(title, guidance) = {
       description: 'Enter your own answer or select from suggestions',
       max_length: 100,
       mandatory: false,
-      suggestions: { url: '{suggestions_url_root}/national-identities.json' },
+      suggestions: { url: '{suggestions_url_root}/national-identities.json', allow_multiple: true },
```

Comment:
Could these be wrapped on multiple lines for readability?. Jsonnet linter isn't very strict.

Reply:
Done, thanks!